docs/usage/getting-started/use-cases.md

```diff
@@ -0,0 +1,135 @@
+# Use Cases
+
+This page describes some common use cases for Renovate, for those who are new and would like to get their heads around the capabilities.
+
+## Development dependency updates
+
+The original use case, and still the most popular one, is for developers to automate dependency updating in their software projects.
+
+### Updating of package files
```

Comment:
This section doesn't mention Renovate's monorepo support, which I think is one of the main differentiators for us vs Dependabot. (Depfu works with monorepos as well)

Might be good to write a small blurb about that here?

Reply:
I think Dependabot has decent monorepo support? But yes, perhaps adding a section on monorepo/multi-language support can be useful